storops/unity/resource/filesystem.py

```diff
@@ -103,6 +103,8 @@ def modify(self, size=None, is_thin=None, tiering_policy=None,
             params['cifsFsParameters'] = cifs_fs_parameters
         if description is not None:
             params['description'] = description
+        if snapScheduleParameters:
```

Comment:
snapScheduleParameters need to change to snap_schedule_parameters

Reply:
Done